<html><head><title>Additional output created by MACS (MACS2_in_Galaxy)</title></head><body><h3>Additional Files:</h3><p><ul>
<li><a href="MACS2_in_Galaxy_model.pdf">MACS2_in_Galaxy_model.pdf</a></li>
<li><a href="MACS2_in_Galaxy_model.r">MACS2_in_Galaxy_model.r</a></li>
<li><a href="MACS2_in_Galaxy_model.r.log">MACS2_in_Galaxy_model.r.log</a></li>
<li><a href="MACS2_in_Galaxy_peaks.narrowPeak">MACS2_in_Galaxy_peaks.narrowPeak</a></li>
<li><a href="MACS2_in_Galaxy_peaks.xls">MACS2_in_Galaxy_peaks.xls</a></li>
<li><a href="MACS2_in_Galaxy_summits.bed">MACS2_in_Galaxy_summits.bed</a></li>
</ul></p>
<h3>Messages from MACS:</h3>
<p><pre>INFO  @ Tue, 20 May 2014 14:24:36: 
# Command line: callpeak -t /groups/csf-ngs/wfsys/galaxy/database/files/047/dataset_47965.dat --format=BAM --name=MACS2_in_Galaxy --gsize=2700000000 --bw=300 --qvalue=0.05 --mfold 10 30
# ARGUMENTS LIST:
# name = MACS2_in_Galaxy
# format = BAM
# ChIP-seq file = ['/groups/csf-ngs/wfsys/galaxy/database/files/047/dataset_47965.dat']
# control file = None
# effective genome size = 2.70e+09
# band width = 300
# model fold = [10, 30]
# qvalue cutoff = 5.00e-02
# Larger dataset will be scaled towards smaller dataset.
# Range for calculating regional lambda is: 10000 bps
# Broad region calling is off
 
INFO  @ Tue, 20 May 2014 14:24:36: #1 read tag files... 
INFO  @ Tue, 20 May 2014 14:24:36: #1 read treatment tags... 
INFO  @ Tue, 20 May 2014 14:24:41:  1000000 
INFO  @ Tue, 20 May 2014 14:24:46:  2000000 
INFO  @ Tue, 20 May 2014 14:24:51:  3000000 
INFO  @ Tue, 20 May 2014 14:24:56:  4000000 
INFO  @ Tue, 20 May 2014 14:25:01:  5000000 
INFO  @ Tue, 20 May 2014 14:25:06:  6000000 
INFO  @ Tue, 20 May 2014 14:25:11:  7000000 
INFO  @ Tue, 20 May 2014 14:25:17:  8000000 
INFO  @ Tue, 20 May 2014 14:25:22:  9000000 
INFO  @ Tue, 20 May 2014 14:25:27:  10000000 
INFO  @ Tue, 20 May 2014 14:25:33:  11000000 
INFO  @ Tue, 20 May 2014 14:25:38:  12000000 
INFO  @ Tue, 20 May 2014 14:25:44:  13000000 
INFO  @ Tue, 20 May 2014 14:25:53:  14000000 
INFO  @ Tue, 20 May 2014 14:25:58:  15000000 
INFO  @ Tue, 20 May 2014 14:26:03:  16000000 
INFO  @ Tue, 20 May 2014 14:26:09:  17000000 
INFO  @ Tue, 20 May 2014 14:26:16:  18000000 
INFO  @ Tue, 20 May 2014 14:26:24:  19000000 
INFO  @ Tue, 20 May 2014 14:26:31:  20000000 
INFO  @ Tue, 20 May 2014 14:26:37:  21000000 
INFO  @ Tue, 20 May 2014 14:26:43:  22000000 
INFO  @ Tue, 20 May 2014 14:26:48:  23000000 
INFO  @ Tue, 20 May 2014 14:26:54:  24000000 
INFO  @ Tue, 20 May 2014 14:26:59:  25000000 
INFO  @ Tue, 20 May 2014 14:27:05:  26000000 
INFO  @ Tue, 20 May 2014 14:27:10:  27000000 
INFO  @ Tue, 20 May 2014 14:27:16:  28000000 
INFO  @ Tue, 20 May 2014 14:27:22:  29000000 
INFO  @ Tue, 20 May 2014 14:27:28:  30000000 
INFO  @ Tue, 20 May 2014 14:27:34:  31000000 
INFO  @ Tue, 20 May 2014 14:27:40:  32000000 
INFO  @ Tue, 20 May 2014 14:27:45:  33000000 
INFO  @ Tue, 20 May 2014 14:27:50:  34000000 
INFO  @ Tue, 20 May 2014 14:27:55:  35000000 
INFO  @ Tue, 20 May 2014 14:28:06:  36000000 
INFO  @ Tue, 20 May 2014 14:28:12:  37000000 
INFO  @ Tue, 20 May 2014 14:28:17:  38000000 
INFO  @ Tue, 20 May 2014 14:28:22:  39000000 
INFO  @ Tue, 20 May 2014 14:28:27:  40000000 
INFO  @ Tue, 20 May 2014 14:28:33:  41000000 
INFO  @ Tue, 20 May 2014 14:28:35: #1 tag size is determined as 43 bps 
INFO  @ Tue, 20 May 2014 14:28:35: #1 tag size = 43 
INFO  @ Tue, 20 May 2014 14:28:35: #1  total tags in treatment: 41076349 
INFO  @ Tue, 20 May 2014 14:28:35: #1 user defined the maximum tags... 
INFO  @ Tue, 20 May 2014 14:28:35: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) 
INFO  @ Tue, 20 May 2014 14:28:49: #1  tags after filtering in treatment: 8649142 
INFO  @ Tue, 20 May 2014 14:28:49: #1  Redundant rate of treatment: 0.79 
INFO  @ Tue, 20 May 2014 14:28:49: #1 finished! 
INFO  @ Tue, 20 May 2014 14:28:49: #2 Build Peak Model... 
INFO  @ Tue, 20 May 2014 14:28:55: #2 number of paired peaks: 569 
WARNING @ Tue, 20 May 2014 14:28:55: Fewer paired peaks (569) than 1000! Model may not be build well! Lower your MFOLD parameter may erase this warning. Now I will use 569 pairs to build model! 
INFO  @ Tue, 20 May 2014 14:28:55: start model_add_line... 
INFO  @ Tue, 20 May 2014 14:28:56: start X-correlation... 
INFO  @ Tue, 20 May 2014 14:28:56: end of X-cor 
INFO  @ Tue, 20 May 2014 14:28:56: #2 finished! 
INFO  @ Tue, 20 May 2014 14:28:56: #2 predicted fragment length is 315 bps 
INFO  @ Tue, 20 May 2014 14:28:56: #2 alternative fragment length(s) may be -464,-259,-223,-185,-100,-49,-38,-2,35,315,364 bps 
INFO  @ Tue, 20 May 2014 14:28:56: #2.2 Generate R script for model : MACS2_in_Galaxy_model.r 
INFO  @ Tue, 20 May 2014 14:28:56: #3 Call peaks... 
INFO  @ Tue, 20 May 2014 14:28:56: #3 Pre-compute pvalue-qvalue table... 
INFO  @ Tue, 20 May 2014 14:30:04: #3 Call peaks for each chromosome... 
INFO  @ Tue, 20 May 2014 14:32:02: #4 Write output xls file... MACS2_in_Galaxy_peaks.xls 
INFO  @ Tue, 20 May 2014 14:32:03: #4 Write peak in narrowPeak format file... MACS2_in_Galaxy_peaks.narrowPeak 
INFO  @ Tue, 20 May 2014 14:32:04: #4 Write summits bed file... MACS2_in_Galaxy_summits.bed 
INFO  @ Tue, 20 May 2014 14:32:04: Done! 
</pre></p>
</body></html>
