<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Frameset//EN" "http://www.w3.org/TR/html4/frameset.dtd">
<html><head>
  <title>GALAXY Workflows - menu</title>
  
  
  <style type="text/css">
  pre {
    white-space: pre-wrap; /* css-3 */
    white-space: -moz-pre-wrap !important; /* Mozilla, since 1999 */
    white-space: -pre-wrap; /* Opera 4-6 */
    white-space: -o-pre-wrap; /* Opera 7 */
    word-wrap: break-word; /* Internet Explorer 5.5+ */
  }
  body{
 font-family: Arial, Verdana, Helvetica, sans-serif;
 font-size: 13px;
 }
a:link {color:#003399;}      /* unvisited link */
a:visited {color:#003399;}  /* visited link */
a:hover {color:#003399;}  /* mouse over link */
a:active {color:#003399;}  /* selected link */ 

ul
{
list-style-type: circle;
padding: 20px;
margin: 0px;
}
ul li
{
padding-left: 5px;
padding-top: 5px;
}

  </style>
  
 
        
        
</head><body>
<p>&nbsp;</p>

<table width="340">


  <tbody>
    <tr>
      <td><b>GALAXY ESSENTIALS</b></td>
    </tr>
    <tr>
      <td>» <a href="main.html#a0" target="data"><b>Prerequisites</b></a><br> &nbsp;&nbsp;&nbsp;Forskalle, QUEUE, GALAXY</td>
    </tr>
    <tr>
      <td>» <a href="main.html#a1" target="data"><b>File Formats</b></a></td>
    </tr>
    <tr>
      <td>» <a href="main.html#a2" target="data"><b>Annotation files</b></a></td>
    </tr>
    <tr>
      <td>» <a href="main.html#a3" target="data"><b>How to get your
data to GALAXY</b></a></td>
    </tr>
    <tr>
      <td>» <a href="demos.html" target="data"><b>Demos</b></a></td>
    </tr>
    <tr>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td><b><a href="faq.html#a4" target="data">Errors - First Aid / FAQ</a></b></td>
    </tr>
    <tr>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td><b>GALAXY-TOOLS &amp; WORKFLOWS</b></td>
    </tr>
    <tr>
      <td>» <a href="main.html#a5" target="data"><b>Converting a
BAM-file to a BED-file</b></a></td>
    </tr>
    <tr>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>» <a href="main.html#a6" target="data"><b>RNA-Seq Analysis
workflow</b></a> <br>
      <ul>
<li><span style="font-weight: bold;">Prerequisites</span>:<br>
        - <a href="main.html#a7" target="data">1. <span style="font-weight: bold;">Count</span> number of
read per gene</a><br>
        - <a href="main.html#a8" target="data">2. <span style="font-weight: bold;">Merge</span> results from
several counts</a></li>
        <li><span style="font-weight: bold;">Statistical Tests for Differences</span>:<br>
- <a href="main.html#a10" target="data">between 2 groups of samples (pairwise)
          </a><br>
- <a href="main.html#a13" target="data">between several groups of samples (ANOVA)</a><br>
- <a href="main.html#a12" target="data">VST and Moderated Fold Changes</a><br>

- Relationships between samples based on <br>&nbsp;&nbsp;a set of most variable genes (PCA):<br>
&nbsp; - <a href="main.html#a15" target="data">a. <span style="font-weight: bold;">PCA</span> Overview</a><br>
&nbsp; - <a href="main.html#a16" target="data">b. <span style="font-weight: bold;">PCA</span> detailed
Analysis on results of</a> </li>
 <li><span style="font-weight: bold;">Descriptive Statistics</span>:<br>
- <a href="main.html#a9" target="data">Matrix Scatterplot</a><br>
- <a href="main.html#a11" target="data">Calculate gene
expression values (<span style="font-weight: bold;">RPKM</span>s)</a></li>

      </ul>
      </td>
    </tr>
<tr>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>» <a href="main.html#a17" target="data"><b>Chip-Seq Analysis
workflow</b></a> <br>
      <ul>
        <li><span style="font-weight: bold;">Peak calling</span>:<br>
- <a href="main.html#a18" target="data">Peak calling (MACS2)</a><br>
- <a href="main.html#a20" target="data">Radar chart for MACS
based peak parameters</a><br>
- <a href="main.html#a21" target="data">Annotate peaks (CEAS)</a><br>
- <a href="main.html#a57" target="data">Peak Annotation, Comparison and Visualization</a><br>
- <a href="main.html#a22" target="data">Compare Lists</a><br>
- <a href="main.html#a31" target="data">Retrieve best peaks</a></li>
<li><span style="font-weight: bold;">Multiple overlaps</span>:<br>
- <a href="main.html#a23" target="data">Multiple overlaps of
peaks (Multovl) </a><br>
- <a href="main.html#a24" target="data">Extract ancestor
track from Multovl output</a><br>
- <a href="main.html#a25" target="data">Multiple overlaps of
peaks, enhanced output</a></li>
        <li><span style="font-weight: bold;">Density-Heatmap</span>:<br>
- <a href="main.html#a26" target="data">Read density over peaks</a><br>
- <a href="main.html#a27" target="data">Plotting for peaks</a><br>
- <a href="main.html#a28" target="data">Read density over transcripts</a><br>
- <a href="main.html#a29" target="data">Plotting for transcripts</a></li>
<li><a href="main.html#a53" target="data">Creating random subsets</a>
        </li>
      </ul>
      </td>
    </tr>
    
   
    
    
    <tr>
      <td>» <a href="main.html#a36" target="data"><b>Graphical
Visualisation</b></a>
      <ul>
        <li><a href="main.html#a37" target="data">Matrix Scatterplot</a></li>
        <li><a href="main.html#a38" target="data">Island Scatterplot</a></li>
        <li><a href="main.html#a39" target="data">Correlation</a></li>
        <li><a href="main.html#a58" target="data">Plotting Genomic Regions (TSS, TES, ...)</a></li>
	<li><a href="main.html#a35" target="data">PLOTTING PROFILES with deeptools</a><br>- compare and normalise bam file(s)<br>- compute Matrix<br>- plot Heatmap<br>- plot Profile</li>

      </ul>
      </td>
    </tr>

<tr>
      <td>» <a href="main.html#a55" target="data"><b>Other useful tools</b></a>
      <ul>
        <li><a href="main.html#a56" target="data">Creating random subsets</a></li>
      </ul>
      </td>
    </tr>


<tr>
      <td>&nbsp;</td>
    </tr>
<tr>
      <td>&nbsp;</td>
    </tr>
<tr>
      <td>» <a href="about.html" target="data"><b>About this page / Acknowledgments</b></a>
      </td>
    </tr>
  </tbody>
</table>
<p>&nbsp;</p>

</body></html>
